sdk/nodejs/src/client.ts

```diff
@@ -280,6 +295,43 @@ export class DevCycleClient {
         this.eventQueue.queueEvent(populatedUser, event)
     }
 
+    /**
+     * Call this to obtain a config that is suitable for use in the "bootstrapConfig" option of client-side JS SDKs
+     * Useful for serverside-rendering use cases where the server performs the initial rendering pass, and provides it
+     * to the client along with the DevCycle config to allow hydration
+     * @param user
+     * @param userAgent
+     */
+    async getClientBootstrapConfig(
+        user: DevCycleUser,
+        userAgent: string,
+    ): Promise<BucketedUserConfig> {
+        const incomingUser = castIncomingUser(user)
+
+        await this.onInitialized
+
+        if (!this.clientConfigHelper) {
+            throw new Error(
+                'enableClientBootstrapping option must be set to true to use getClientBootstrapConfig',
+            )
+        }
+
+        try {
+            const { generateClientPopulatedUser } = await import(
```

Comment:
Can you explain the dynamic import here?

Reply:
basically those who are doing bootstrapping on the server would need to have the js client SDK installed as well so that it can generate the DVCPopulatedUser correctly, and if they're doing bootstrapping they'd already have to have it installed anyway